dynamiqs/mesolve/mesolve.py

```diff
@@ -56,7 +63,16 @@ def mesolve(
     f = compute_vmap(_mesolve, options.cartesian_batching, is_batched, out_axes)
 
     # === apply vectorized function
-    return f(H, jump_ops, rho0, tsave, exp_ops, solver, gradient, options)
+    return f(
+        H,
+        jump_ops,
+        rho0,
+        tsave,
+        exp_ops,
+        solver,
+        gradient,
+        options,
```

Comment:
```suggestion
        options
```
and black will format back to a single line.